internal/service/advancedcluster/fw_common_advanced_cluster.go

```diff
@@ -0,0 +1,129 @@
+package advancedcluster
```

Comment:
This file will be renamed to common_advanced_cluster.go after migration of resources complete